src/main/java/com/google/api/codegen/util/SymbolTable.java

```diff
@@ -87,6 +87,11 @@ public String getNewSymbol(String desiredName) {
     return desiredName + suffix;
   }
 
+  /** Returns true if a {@code symbol} is already used. */
+  public boolean isSymbolUsed(Name symbol) {
```

Comment:
The `SymbolTable` deals only with symbols, I think we can just call this `isUsed` here.

Reply:
Maybe so that it reads better in English it should be `SymbolTable.uses(symbol)` or `SymbolTable.contains(symbol)'?

Reply:
Good idea!